### ISSUE TITLE
Tenant is required

### ISSUE THREAD

**Issue by talfirevic** [IdentityManager/IdentityManager.MembershipReboot#10]
When trying to add new role

MembershipRebootIdentityManagerService.cs catches exception on 
var group = this.groupService.Create(name);

with message Tenant is required.

**Comment by brockallen** [comment]
Which release are you on? IIRC, this has been fixed on dev.

**Comment by talfirevic** [comment]
v1-beta-3

**Comment by brockallen** [comment]
Does your group service do something like this:

https://github.com/IdentityManager/Thinktecture.IdentityManager.MembershipReboot/blob/master/source/Host/Config/CustomGroup.cs#L37

**Comment by talfirevic** [comment]
Well, no.

In the source I've downloaded constructor signature is with repo parameter only.

Yeah I've downloaded my source from "Releases" section and that source doesn't include fixes from Jan 14.

b5c54099ec14cd8e8e5c2eb87cdd9aa064eabb55
adb3789f435f2254b8be690b90d51fa64babca80
1cc2b96f1e0a9f9930c986e603e83025468210ec

Getting the latest version did the trick.

Perhaps release source should include those three commits too :)
